content/docs/getting_started/install_apps.md

```diff
@@ -0,0 +1,144 @@
+---
+title: "Deploy Sample Applications"
+description: "Deploy the sample bookstore applications"
+type: docs
+weight: 2
+aliases: = ["/docs/install/manual_demo/"]
+---
+
+## Deploy Applications
+
+In this section we will deploy 5 different Pods, and we will apply policies to control the traffic between them.
+
+- `bookbuyer` is an HTTP client making requests to `bookstore-v1`. This traffic is **permitted**.
+- `bookthief` is an HTTP client and much like `bookbuyer` also makes HTTP requests to `bookstore`. This traffic should be **blocked**.
+- `bookstore-v1` is a server, which responds to HTTP requests. It is also a client making requests to the `bookwarehouse` service.
```

Comment:
It seems like the demo workflow is being updated. Previously the demo walked through permissive mode first which would deploy the `bookstore` app with the `bookstore-v1` identity, and then deploy bookstore-v2 to showcase traffic splitting with SMI. Isn't that the case anymore? Could you also describe at a high level in the commit messages what changes are being made to the demo?

Reply:
I didn't intentionally make any significant changes to the overall steps of the demo. Since the old manual demo page is now split into multiple files, the order of the steps isn't super clear from just looking at the diffs, but the [preview](https://deploy-preview-195--osm-docs.netlify.app/docs/getting_started/) renders the pages in the right order.

The demo still walks through installing OSM in permissive mode, installs the apps and shows them running, then disables permissive mode and creates SMI policies.

Reply:
It seems to me that there's a difference. Previously, the demo starts with permissive mode which deploys the `bookstore` application where `bookstore.yaml` is used (not `bookstore-v1`):
![image](https://user-images.githubusercontent.com/21697719/135498674-aa0cd438-50b5-4c3c-9934-b2dce51f6728.png)

With your changes, the app has been changed to `bookstore-v1` where `bookstore-v1.yaml` is used:
![image](https://user-images.githubusercontent.com/21697719/135498775-806feb25-14a7-43a1-b184-494ae396843f.png)

I am unsure if this was intentional, but there is a reason why permissive mode used `bookstore` and not `bookstore-v1`. It's fine if we want to change that, but I'd like to see that surfaced in the commit message and the reason for doing so.

Reply:
I essentially renamed the old bookstore.yaml to bookstore-v1.yaml, then added a new bookstore.yaml that's just the root bookstore service. The most significant change here is aligning the manifests closer to the automated demo, which has a bookstore root service and bookstore-v1 and bookstore-v2 backend services. To compensate for this change, I've updated the bookbuyer (and bookthief) to [target the bookstore-v1 service](https://github.com/openservicemesh/osm/pull/4180/files#diff-482c2abe2dc68b82c84b7fc63d58804898a70d49a2ac1f3b503ef08b0a48588eR41) initially, then [update that later](https://github.com/openservicemesh/osm-docs/pull/195/files#diff-91a064e053df3695c52c91b719f85244b867e4f3e3f13ff3a0d6847c11f68e48R24-R29) to point to the bookstore root service once that's been created. Does this address what you mean by "there is a reason why permissive mode used `bookstore` and not `bookstore-v1`"?

Reply:
The goal should not be to align the manual demo to the automated demo, because the automated demo is not user-friendly. In the past, I had worked to address this issue by reworking the manual demo so that the onboarding experience for users is clear, which is not the case with the automated demo (see https://github.com/openservicemesh/osm/commit/54b7749edd7e3b12ecfca403805e1b581c4c8a6a).

The [update that later section](https://github.com/openservicemesh/osm-docs/pull/195/files#diff-91a064e053df3695c52c91b719f85244b867e4f3e3f13ff3a0d6847c11f68e48R24-R29) does not make sense to me. Users aren't going to be able to update their application binary in a way that our demo manifests allow. We need to be able to demonstrate the following in this order:
1. Install OSM
2. Enable permissive traffic policy mode
3. Deploy apps
4. Demonstrate that bookbuyer and bookthief apps can access the `bookstore-v1` service or `bookstore` service. If the `bookstore` service is used, then there must be a pod backing the `bookstore` service for permissive mode to function.
5. Switch to SMI mode and verify that all traffic is blocked
6. Deploy TrafficTarget to allow bookbuyer to access bookstore, and showcase that bookthief cannot access bookstore
7. Document TrafficSplit usage. The recommended way should be to deploy a second version of the app while the first app is still running. If this isn't possible, then it must be called out. Requiring users to update thier app binaries to use traffic split is a serious caveat, so if we can demo this in a way where it's avoidable, we should explore that.

Reply:
> Users aren't going to be able to update their application binary in a way that our demo manifests allow.

This is true, but the steps I added only involve changing an environment variable on the bookbuyer and bookthief containers.

These changes still adhere to the steps you describe AFAICT. Where do you see that they don't?

Reply:
I did not suggest that the doc steps don't adhere to the steps described above, but renaming bookstore -> bookstore-v1 in the docs and environment variables seems confusing enough to the extent that I suggest making that change in a separate PR. I'd really like for this PR to only refactor the getting-started page, not the demo docs. If you want to make changes to the demo docs, let's carve it out and make it a separate change so we can clearly demonstrate caveats with respect to changing the request URLs. Some of the problems I am highlighting might be present in the existing version of the docs as well, which I think must be clearly documented for v1.0.

Reply:
We've decided not to go forward with the bookstore->bookstore-v1 changes. Those specific changes have been removed.